github/repos_autolinks.go

```diff
@@ -0,0 +1,98 @@
+// Copyright 2021 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package github
+
+import (
+	"context"
+	"fmt"
+)
+
+// AutolinkOptions specifies parameters for RepositoriesService.AddAutolink method
+// Also this struct is embedded into an Autolink struct which contains one additional ID field
```

Comment:
Let's please delete this line and duplicate the fields in the `Autolink` struct below.
This will be more consistent with the rest of the repo.

Reply:
Line 14 can be deleted.

Reply:
Done.